packages/vite-plugin-cloudflare/src/cloudflare-environment.ts

```diff
@@ -140,28 +140,6 @@ export function createCloudflareEnvironmentOptions(
 				createEnvironment(name, config) {
 					return new CloudflareDevEnvironment(name, config);
 				},
-				optimizeDeps: {
```

Comment:
Vite's moved this option to the top level: https://github.com/vitejs/vite/pull/18465